low-level/sspl-ll.spec

```diff
@@ -78,7 +78,9 @@ cp -p $SSPL_SETUP/consuldump.py $SSPL_BASE/low-level/
 # take backup of cache folder if exists
 mkdir -p /opt/seagate/backup/%{version}
 [ -f /etc/sspl.conf ] && cp -p /etc/sspl.conf /opt/seagate/backup/%{version}/sspl.conf
-[ -d /var/%{product_family}/sspl ] && cp -Rp /var/%{product_family}/sspl /opt/seagate/backup/%{version}/
+if [ -d /var/%{product_family}/sspl ]; then
```

Comment:
logic is same as existing code, what different is being achieved here? Why this is failing installation suddenly ?

Reply:
If condition won't be satisfied, it gives rc 1 for single line bash.

> [root@ssc-vm-3120 ~]# [ -d /var/cortx/sspl ] && echo "hi"
> [root@ssc-vm-3120 ~]#
> [root@ssc-vm-3120 ~]# echo $?
> 1

Same condition but this gives rc 0.

> [root@ssc-vm-3120 ~]# if [ -d /var/cortx/sspl ]; then
> > echo "hi"
> > fi
> [root@ssc-vm-3120 ~]#
> [root@ssc-vm-3120 ~]# echo $?
> 0